### ISSUE TITLE
Ignoring tests

### ISSUE THREAD

**Issue by wildart** [IainNZ/BaseTestNext.jl#11]
Sometimes, you write tests in advance of implementation, and you want them not to not blow your test results. It would be nice to provide a construct for ignoring tests, while they got reported at the test report. Something like:

``` julia
@ignore @test 1=1
```

Show ignored  tests in a successfully terminated test report

```
Test Summary:  | Pass  Ignored Total 
Some Tests     |    4        1     5
```

or in a test report with errors

```
Test Summary:  | Pass  Ignored Error Total 
Some Tests     |    3        1     1     5
```

**Comment by tkelman** [comment]
ref https://github.com/JuliaLang/julia/pull/16105 - the intent of this repo is to make the new implementation of Base.Test available on 0.4, and possibly make any future Base.Test changes available on 0.5 release if they don't get backported in 0.5.x.

**Comment by wildart** [comment]
Thanks for ref.
